src/SourceTerms/Deleptonization/CPU_Src_Deleptonization.cpp

```diff
@@ -500,41 +464,41 @@ void Src_End_Deleptonization()
 
 
 //-----------------------------------------------------------------------------------------
-// Function    :  YeOfRhoFunc
+// Function    :  yeofrhofunc
 // Description :  Calculate electron fraction Ye from the given density and
 //                deleptonization parameters
 //
 // Note        :  1. Invoked by Src_Deleptonization()
 //                2. Ref: M. Liebendoerfer, 2005, ApJ, 603, 1042-1051 (arXiv: astro-ph/0504072)
 //
-// Parameter   :  DENS_CGS  : density in CGS from which Ye is caculated
-//             :  DELEP_RHO1: parameter for the parameterized deleptonization fitting formula
-//             :  DELEP_RHO2: parameter for the parameterized deleptonization fitting formula
-//             :  DELEP_YE1 : parameter for the parameterized deleptonization fitting formula
-//             :  DELEP_YE2 : parameter for the parameterized deleptonization fitting formula
-//             :  DELEP_YEC : parameter for the parameterized deleptonization fitting formula
+// Parameter   :  Dens_CGS  : density in CGS from which Ye is caculated
+//             :  Delep_Rho1: parameter for the parameterized deleptonization fitting formula
+//             :  Delep_Rho2: parameter for the parameterized deleptonization fitting formula
+//             :  Delep_Ye1 : parameter for the parameterized deleptonization fitting formula
+//             :  Delep_Ye2 : parameter for the parameterized deleptonization fitting formula
+//             :  Delep_Yec : parameter for the parameterized deleptonization fitting formula
 //
-// Return      :  YeOfRhoFunc
+// Return      :  yeofrhofunc
 //-----------------------------------------------------------------------------------------
 GPU_DEVICE static
-real YeOfRhoFunc( const real DENS_CGS, const real DELEP_RHO1, const real DELEP_RHO2,
-                  const real DELEP_YE1, const real DELEP_YE2, const real DELEP_YEC )
+real yeofrhofunc( const real Dens_CGS, const real Delep_Rho1, const real Delep_Rho2,
+                  const real Delep_Ye1, const real Delep_Ye2, const real Delep_Yec )
 {
 
    real XofRho, Ye;
 
-   XofRho = (  2.0 * LOG10( DENS_CGS ) - LOG10( DELEP_RHO2 ) - LOG10( DELEP_RHO1 )  )
-          / (  LOG10( DELEP_RHO2 ) - LOG10( DELEP_RHO1 )  );
-   XofRho = MAX( -1.0, MIN( 1.0, XofRho ) );
+   XofRho = LOG10( SQR( Dens_CGS ) / Delep_Rho2 / Delep_Rho1 )
```

Comment:
`/ ( Delep_Rho2 * Delep_Rho1 )`

Reply:
I changed it to `/ ( Delep_Rho2 * Delep_Rho1 )`